From f400ad7c851ab520b6b3248268b475ae36ade3e8 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Fri, 31 May 2019 19:33:25 +0000
Subject: [PATCH] TRITON-1711 update volapi base image to
 triton-origin-x86_64-18.4.0 TRITON-1712 volapi should use use node v6
 Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 Makefile     | 10 +++++-----
 package.json |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 16ea614..bd7b531 100644
--- a/Makefile
+++ b/Makefile
@@ -26,12 +26,11 @@ JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 NAME = volapi
 SMF_MANIFESTS_IN = smf/manifests/$(NAME)-server.xml.in smf/manifests/$(NAME)-updater.xml.in
 
-NODE_PREBUILT_VERSION=v4.9.0
+NODE_PREBUILT_VERSION=v6.17.0
 
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_TAG=zone
-	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
-	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_TAG=zone64
+	NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 ENGBLD_USE_BUILDIMAGE	= true
@@ -51,7 +50,8 @@ ROOT            := $(shell pwd)
 RELEASE_TARBALL := $(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR     := /tmp/$(NAME)-$(STAMP)
 
-BASE_IMAGE_UUID = 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
+# our base image is triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SDC Volumes API
 AGENTS		= amon config registrar
diff --git a/package.json b/package.json
index e928088..9825604 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sdc-volapi",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "SDC's Volumes API",
   "main": "server.js",
   "scripts": {
-- 
2.21.0

